### PR TITLE
OCPBUG#10557: Added note related to VMware vSphere CSI driver operator

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.11+

Issue:
[OCPBUG 10557](https://issues.redhat.com/browse/OCPBUGS-10557)

Link to docs preview:
[Doc Preview](https://60960--docspreview.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-vsphere.html#vsphere-csi-driver-reqs_persistent-storage-csi-vsphere)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/vmware-csi-driver-reqs.adoc [hunk 1]
```diff
@@ -29,4 +29,9 @@ To install the vSphere CSI Driver Operator, the following requirements must be m
 * Virtual machines of hardware version 15 or later
 * No third-party vSphere CSI driver already installed in the cluster
 
-If a third-party vSphere CSI driver is present in the cluster, {product-title} does not overwrite it. The presence of a third-party vSphere CSI driver prevents {product-title} from upgrading to {product-title} 4.13 or later.
\ No newline at end of file
+If a third-party vSphere CSI driver is present in the cluster, {product-title} does not overwrite it. The presence of a third-party vSphere CSI driver prevents {product-title} from upgrading to {product-title} 4.13 or later.
+
+[NOTE]
+====
+VMware vSphere CSI Driver Operator will only work on vSphere clusters.
```
Comment (duanwei33) [inline modules/vmware-csi-driver-reqs.adoc:36]: How about 
```suggestion
The VMware vSphere CSI Driver Operator  is only supported on OpenShift when the cluster has been deployed with `platform: vsphere` in the install manifest.```

Comment (duanwei33) [reply modules/vmware-csi-driver-reqs.adoc:36]: cc @gnufied if you have better idea :)

Comment (subhtk) [reply modules/vmware-csi-driver-reqs.adoc:36]: Ack.

File: modules/vmware-csi-driver-reqs.adoc [hunk 2]
```diff
@@ -29,4 +29,9 @@ To install the vSphere CSI Driver Operator, the following requirements must be m
 * Virtual machines of hardware version 15 or later
 * No third-party vSphere CSI driver already installed in the cluster
 
-If a third-party vSphere CSI driver is present in the cluster, {product-title} does not overwrite it. The presence of a third-party vSphere CSI driver prevents {product-title} from upgrading to {product-title} 4.13 or later.
\ No newline at end of file
+If a third-party vSphere CSI driver is present in the cluster, {product-title} does not overwrite it. The presence of a third-party vSphere CSI driver prevents {product-title} from upgrading to {product-title} 4.13 or later.
+
+[NOTE]
+====
+VThe VMware vSphere CSI Driver Operator  is only supported on OpenShift when the cluster has been deployed with `platform: vsphere` in the install manifest.
```
Comment (duanwei33) [inline modules/vmware-csi-driver-reqs.adoc:36]:
```suggestion
The VMware vSphere CSI Driver Operator is only supported on OpenShift when the cluster has been deployed with `platform: vsphere` in the install manifest.
```

Comment (subhtk) [reply modules/vmware-csi-driver-reqs.adoc]: Ack

File: modules/vmware-csi-driver-reqs.adoc [hunk 3]
```diff
@@ -29,4 +29,9 @@ To install the vSphere CSI Driver Operator, the following requirements must be m
 * Virtual machines of hardware version 15 or later
 * No third-party vSphere CSI driver already installed in the cluster
 
-If a third-party vSphere CSI driver is present in the cluster, {product-title} does not overwrite it. The presence of a third-party vSphere CSI driver prevents {product-title} from upgrading to {product-title} 4.13 or later.
\ No newline at end of file
+If a third-party vSphere CSI driver is present in the cluster, {product-title} does not overwrite it. The presence of a third-party vSphere CSI driver prevents {product-title} from upgrading to {product-title} 4.13 or later.
```
Comment (skopacz1) [inline modules/vmware-csi-driver-reqs.adoc:32]: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update

Update is preferred over upgrade for consistency in our docs:

```suggestion
If a third-party vSphere CSI driver is present in the cluster, {product-title} does not overwrite it. The presence of a third-party vSphere CSI driver prevents {product-title} from updating to {product-title} 4.13 or later.
```

File: modules/vmware-csi-driver-reqs.adoc [hunk 4]
```diff
@@ -29,4 +29,9 @@ To install the vSphere CSI Driver Operator, the following requirements must be m
...
+
+[NOTE]
+====
+The VMware vSphere CSI Driver Operator  is only supported on OpenShift when the cluster has been deployed with `platform: vsphere` in the install manifest.
```
Comment (skopacz1) [inline modules/vmware-csi-driver-reqs.adoc:36]: It renders fine in the preview, but there's an extra space after "Operator" in this note.

Comment (skopacz1) [inline modules/vmware-csi-driver-reqs.adoc:36]: Suggestion for minimalism and closer placement of "only":

```suggestion
The VMware vSphere CSI Driver Operator is supported only on clusters deployed with `platform: vsphere` in the install manifest.
```

**Review by skopacz1 — COMMENTED** [review]
A few nits but otherwise looks good! I am new to peer review, so please wait until @adellape approves of my review before implementing any of my feedback. Thank you!

**Review by adellape — COMMENTED** [review]
@skopacz1 @subhtk Peer review looks good 👍 Just one added comment.

File: modules/vmware-csi-driver-reqs.adoc (hunk 4 above)
Comment (subhtk) [reply modules/vmware-csi-driver-reqs.adoc:36]: Ack

Comment (subhtk) [reply modules/vmware-csi-driver-reqs.adoc:36]: Ack.

[the remark this answers is not in the document]
Comment (subhtk) [reply modules/vmware-csi-driver-reqs.adoc:36]: I will go with "installation manifest" too. Ack.

File: modules/vmware-csi-driver-reqs.adoc (hunk 3 above)
Comment (subhtk) [reply modules/vmware-csi-driver-reqs.adoc:32]: ack

**Pull request merged**